packages/react-component-library/src/components/RangeSlider/RangeSlider.test.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import '@testing-library/jest-dom/extend-expect'
+import { render, RenderResult } from '@testing-library/react'
+
+import { IconBrightnessLow, IconBrightnessHigh } from '@royalnavy/icon-library'
+import { RangeSlider } from '.'
+
+describe('RangeSlider', () => {
+  let wrapper: RenderResult
+  let mode: number
+  let domain: number[] // lower and upper bounds
+  let values: number[] // initial handle values
+  let step: number
+  let tickCount: number
+
+  describe('single handle, unstepped, with callbacks', () => {
+    beforeEach(() => {
+      mode = 1
+      domain = [0, 40]
+      values = [20]
+      step = 10
+      tickCount = 4
+
+      wrapper = render(
+        <RangeSlider
+          mode={mode}
+          domain={domain}
+          values={values}
+          step={step}
+          tickCount={tickCount}
+        />
+      )
+    })
+
+    it('should render a single handle', () => {
+      expect(wrapper.queryAllByTestId('rangeslider-handle').length).toBe(1)
+    })
+
+    it('should apply the correct classes', () => {
+      expect(
+        wrapper
+          .queryByTestId('rangeslider')
+          .classList.contains('rn-rangeslider')
+      ).toBe(true)
```

Comment:
There is also `toContains` which might provide a better error message if it fails.